src/app/components/PageLayoutWrapper/index.tsx

```diff
@@ -33,15 +33,11 @@ type Props = {
   pageData: {
     metadata: {
       type: PageTypes;
-      topics?: [
-        {
-          topicName: string;
-        },
-      ];
-    };
-    content?: {
-      model?: ModelType;
+      topics?: { topicName: string }[];
     };
+    content?: { model?: ModelType };
+    secondaryColumn?: { topStories: any[] };
+    mostRead?: { items: any[] };
```

Comment:
Very minor since its an experiment, but we do have types for TopStories and MostRead in https://github.com/bbc/simorgh/blob/7c89980c70d18ffa909aa6cfeba0c97887a2e5bb/src/app/models/types/optimo.ts#L128-L153

Reply:
I have now used pre-existing types in the code 👍